hail/python/hailtop/aiotools/time_limited_max_size_cache.py

```diff
@@ -2,20 +2,29 @@
 
 import time
 import asyncio
+import prometheus_client as pc  # type: ignore
+from prometheus_async.aio import time as prom_async_time  # type: ignore
```

Comment:
On further inspection, hailtop doesn't depend on prometheus (it's in `docker/requirements.txt` which is for services code). We might need to move this into gear, I've started a thread in zulip.